apps/directory/src/stores/filtersStore.ts

```diff
@@ -6,8 +6,10 @@ import { useBiobanksStore } from "./biobanksStore";
 import { useSettingsStore } from "./settingsStore";
 import { useCheckoutStore } from "./checkoutStore";
 import { applyBookmark, createBookmark } from "../functions/bookmarkMapper";
+//@ts-ignore
```

Comment:
why is the ignore needed here ?

Reply:
Because the molgenis components main file doesn't expose typings, which TS doesn't like.